deploy/restartComponent.sh

```diff
@@ -3,16 +3,17 @@
 HOST=`hostname`
 DATENOW=`date +%s`
 
+# Get a few environment variables in
```

Comment:
please add here which env variable it provides since it is unclear in a script where `$install` and `$manage` comes from (as I bet they come from this `env.sh` script).